dimod/generators/wireless.py

```diff
@@ -0,0 +1,985 @@
+# Copyright 2023 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express 93or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from collections import namedtuple
+from functools import wraps
+from itertools import product
+from typing import Literal, Optional, Tuple, Union
+
+import numpy as np
+
+import dimod
+
+__all__ = ['mimo', 'comp', ]
```

Comment:
`mimo` is OK, but `comp` is quite overloaded. [The wikipedia page](https://en.wikipedia.org/wiki/Cooperative_MIMO) uses `CO-MIMO` so IMO we should use `co_mimo` or `cooperative_mimo` here. I prefer the latter personally.

Alternatively we could spell both out, but I agree that `multiple_input_and_multiple_output` is quite long.

Reply:
Okay, so just to clarify, ours is [Coordinated multipoint](https://en.wikipedia.org/wiki/Cooperative_MIMO) on that same page. I [don't see](https://en.wikipedia.org/wiki/Comp) that `comp` is overloaded, and would prefer to keep it, but if there is objection, are we okay with `mimo` and `coordinated_multipoint` to avoid the long  `multiple_input_and_multiple_output`? (@jackraymond, @arcondello )

Reply:
No strong opinion, docstring makes sense to me, no name is perfect.

Reply:
I mean overloaded in the sense of when I see `comp` out of context, I think "comparison" or "compilation"

Reply:
Okay, closed in [this commit](https://github.com/dwavesystems/dimod/pull/1357/commits/b00e943aad189ff1590d225877611c06f071d0cd)